presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoNativeIcebergTpcdsQueriesParquetUsingThrift.java

```diff
@@ -35,4 +35,118 @@ protected ExpectedQueryRunner createExpectedQueryRunner()
         this.storageFormat = "PARQUET";
         return PrestoNativeQueryRunnerUtils.createJavaIcebergQueryRunner("PARQUET");
     }
+
+    protected void doDeletes() {}
+    protected void runAllQueries() throws Exception
+    {
+        testTpcdsQ1();
+        testTpcdsQ2();
+        testTpcdsQ3();
+        testTpcdsQ4();
+        testTpcdsQ5();
+        testTpcdsQ6();
+        testTpcdsQ7();
+        testTpcdsQ8();
+        testTpcdsQ9();
+        testTpcdsQ10();
+        testTpcdsQ11();
+        testTpcdsQ12();
+        testTpcdsQ13();
+        testTpcdsQ14_1();
+        testTpcdsQ14_2();
+        testTpcdsQ15();
+        testTpcdsQ16();
+        testTpcdsQ17();
+        testTpcdsQ18();
+        testTpcdsQ19();
+        testTpcdsQ20();
+        testTpcdsQ21();
+        testTpcdsQ22();
+        testTpcdsQ23_1();
+        testTpcdsQ23_2();
+        testTpcdsQ24_1();
+        testTpcdsQ24_2();
+        testTpcdsQ25();
+        testTpcdsQ26();
+        testTpcdsQ27();
+        testTpcdsQ28();
+        testTpcdsQ29();
+        testTpcdsQ30();
+        testTpcdsQ31();
+        testTpcdsQ32();
+        testTpcdsQ33();
+        testTpcdsQ34();
+        testTpcdsQ35();
+        testTpcdsQ36();
+        testTpcdsQ37();
+        testTpcdsQ38();
+        testTpcdsQ39_1();
+        testTpcdsQ39_2();
+        testTpcdsQ40();
+        testTpcdsQ41();
+        testTpcdsQ42();
+        testTpcdsQ43();
+        testTpcdsQ44();
+        testTpcdsQ45();
+        testTpcdsQ46();
+        testTpcdsQ47();
+        testTpcdsQ48();
+        testTpcdsQ49();
+        testTpcdsQ50();
+        testTpcdsQ51();
+        testTpcdsQ52();
+        testTpcdsQ53();
+        testTpcdsQ54();
+        testTpcdsQ55();
+        testTpcdsQ56();
+        testTpcdsQ57();
+        testTpcdsQ58();
+        testTpcdsQ59();
+        testTpcdsQ60();
+        testTpcdsQ61();
+        testTpcdsQ62();
+        testTpcdsQ63();
+        testTpcdsQ65();
+        testTpcdsQ66();
+        testTpcdsQ67();
+        testTpcdsQ68();
+        testTpcdsQ69();
+        testTpcdsQ70();
+        testTpcdsQ71();
+        testTpcdsQ72();
+        testTpcdsQ73();
+        testTpcdsQ74();
+        testTpcdsQ75();
+        testTpcdsQ76();
+        testTpcdsQ77();
+        testTpcdsQ78();
+        testTpcdsQ79();
+        testTpcdsQ80();
+        testTpcdsQ81();
+        testTpcdsQ82();
+        testTpcdsQ83();
+        testTpcdsQ84();
+        testTpcdsQ85();
+        testTpcdsQ86();
+        testTpcdsQ87();
+        testTpcdsQ88();
+        testTpcdsQ89();
+        testTpcdsQ90();
+        testTpcdsQ91();
+        testTpcdsQ92();
+        testTpcdsQ93();
+        testTpcdsQ94();
+        testTpcdsQ95();
+        testTpcdsQ96();
+        testTpcdsQ97();
+        testTpcdsQ98();
+        testTpcdsQ99();
+    }
+
+    protected void dropTables()
+    {
+        for (String table : tpcdsFactTableNames) {
```

Comment:
Why is the implementation in this class? Can any of AbstractTestNativeTpcdsQueries's children have different implementation than this? If not, let's move it to AbstractTestNativeTpcdsQueries's

Reply:
Will move it to base class.